Makefile

```diff
@@ -69,6 +62,12 @@ build-container-linux-%:
 		--build-arg OS=linux \
 		--build-arg ARCH=$* .
 
+build-container-windows-%:
```

Comment:
I remember that @mattcary had problems building a Windows image without pushing it to a registry, did you have any issues running this command?

Reply:
Hi @mauriciopoppe 
I could remove this make `build-container-windows-%` command in Makefile, actually it does not change the default behavior, the Windows Dockerfile build works well now which shows the `OSVersion` refactor change works well:
https://k8s-testgrid.appspot.com/sig-storage-image-build#post-sig-storage-local-static-provisioner-push-images

Reply:
great, thanks for the clarification